src/style/app.module.css

```diff
@@ -3027,38 +3096,72 @@ form > input {
 }
 
 .email_button {
+  --bs-btn-active-bg: var(--search-button-bg);
+  --bs-btn-active-border-color: var(--search-button-bg);
+  --bs-btn-hover-bg: var(--search-button-bg);
+  --bs-btn-hover-border-color: var(--search-button-bg);
   position: absolute;
   z-index: 10;
   bottom: 0;
   right: 0;
   height: 100%;
   display: flex;
   background-color: var(--search-button-bg);
-  border-color: var(--search-button-border, var(--black-color));
+  border-color: var(--search-button-bg);
   justify-content: center;
   align-items: center;
+  color: var(--brown-color);
+}
+.email_button:hover {
+  color: var(--brown-color) !important;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with future attempts to having Dark Mode.
- For example `--search-button-bg` is much better than `--light-blue`